bin/aapp_dr_runner.py

```diff
@@ -59,23 +72,10 @@
 If you change these, you will also have to change the decommutation scripts.
 """
 
-from urllib.parse import urlparse
-import posttroll.subscriber
-from posttroll.publisher import Publish
-from posttroll.message import Message
-from aapp_runner.helper_functions import overlapping_timeinterval
 
-import tempfile
-from glob import glob
 # import os
-import shutil
 # import aapp_stat
-import threading
-from subprocess import Popen, PIPE
-import shlex
 # import subrocess
```

Comment:
Could you remove the unnecessary imports? Also running `isort` would be nice.

Reply:
Yes, think I did...

Reply:
I meant the comment out imports should be removed too :)

Reply:
Oh sorry, overlooked. Done